src/c4v/scraper/spiders/el_pitazo.py

```diff
@@ -0,0 +1,73 @@
+# Internal imports
+import c4v.scraper.utils as utils
+
+# External imports
+import scrapy
+
+# Python imports
+from typing import List, Dict, Any
+
+
+class ElPitazoSpider(scrapy.Spider):
+    """
+        Spider to scrape ElPitazo data 
+    """
+
+    name = "el_pitazo"
+
+    start_urls = []
+
+    def parse(self, response) -> Dict[str, Any]:
+        """
+            Returns a dict like structure with the following 
+            fields:
+                + title
+                + date
+                + categories
+                + body
+                + author 
+                + tags
+        """
+
+        # These are simple properties, just get its text with a valid
+        # selector
+        title = utils.get_element_text(".tdb-title-text", response) or ""
+        date = utils.get_element_text(".entry-date", response) or ""
+        author = utils.get_element_text(".tdb-author-name", response) or ""
+
+        body = self._get_body(response)
+
+        tags = self._get_tags(response)
+
+        # categories
+        categories = response.css(".tdb-entry-category").getall()
+        categories = list(map(utils.strip_http_tags, categories))
+
+        return {
+            "title": title,
+            "date": date,
+            "categories": categories,
+            "body": body,
+            "author": author,
+            "tags": tags,
+        }
+
+    def _get_body(self, response) -> str:
+        """
+            Get article body as a single string
+        """
+        body = response.css("#bsf_rt_marker > p").getall()
```

Comment:
you can use add ` ::text`, to let the query know you only want the text within those elements selected.  By doing this you can, probably, omit the following two lines of code, letting scrapy clean any html tags that may exist within the retrieved body. And somewhat guarantees to have unicode strings (which we are currently not getting).
`body = response.css("#bsf_rt_marker > p ::text").getall()`

This might be done on a different PR, maybe as a bonus on #73.